ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -18,12 +18,17 @@ import scalaz.std.option._
 import scalaz.syntax.show._
 import com.daml.cliopts.{GlobalLogLevel, Logging}
 import com.daml.http.util.Logging.{InstanceUUID, instanceUUIDLogCtx}
+import com.daml.ledger.resources.ResourceContext
 import com.daml.logging.{ContextualizedLogger, LoggingContextOf}
 
+import com.daml.platform.sandbox.metrics.MetricsReporting
```

Comment:
Ah I guess that’s what pulls in `sandbox-common`. Maybe we should move that to a separate package at some point. Doesn’t seem specific to sandbox.